autogpt_platform/frontend/src/components/NavBar.tsx

```diff
@@ -43,7 +44,13 @@ export async function NavBar() {
           <SheetContent side="left">
             <nav className="grid gap-6 text-lg font-medium">
               <Link
-                href="/marketplace"
+                href={
+                  getBehaveAs() === "CLOUD"
+                    ? "/marketplace"
+                    : "http://dev-builder.agpt.co/marketplace"
+                }
+                // open in new tab if not local
+                target={getBehaveAs() === "CLOUD" ? "_self" : "_blank"}
```

Comment:
Is this because of the CORS & next/router ? 
we can do current_host == NEXT_PUBLIC_AGPT_MARKETPLACE_URL.host ?

Reply:
Open in new tab or current